tests/regression/tests/REQUEST-932-APPLICATION-ATTACK-RCE/932237.yaml

```diff
@@ -209,3 +209,19 @@ tests:
             version: HTTP/1.0
           output:
             no_log_contains: id "932237"
+  - test_title: 932237-14
+    desc: "FP with word settings Github Issue #3393"
```

Comment:
Don't reference GH issues. At some point in the future, GH will be replaced with something else and the issue numbers will have lost all meaning.

Reply:
CRS is moving away from github?
I've removed any references to GH issues

Reply:
No no no 😄. I'm just saying that every software product becomes obsolete at some point. Information should not rely on an external system to exist indefinitely.